app/components/windows/SourcesShowcase.vue

```diff
@@ -415,7 +503,7 @@
           <div
             class="source source--widget"
             :class="{'source--active': inspectedSource === widgetTypes.TheJar}"
-            @click="inspectSource(widgetTypes.TheJar)"
+            @click="inspectSource(widgetTypes.TheJar, 'theJar')"
```

Comment:
Does this string here do anything?  As far as I can tell, this function only takes 1 argument.